rasa/shared/core/domain.py

```diff
@@ -1106,7 +1107,7 @@ def _get_featurized_entities(self, latest_message: UserUttered) -> Set[Text]:
 
     def _get_user_sub_state(
         self, tracker: "DialogueStateTracker"
-    ) -> Dict[Text, Union[Text, Tuple[Text]]]:
+    ) -> Dict[Text, Union[None, Text, List[Optional[Text]], Tuple[str, ...]]]:
```

Comment:
What does the `...` represent here?

Reply:
It means that it can be a tuple of varying length, but all of the items will be of type `str`. See: https://mypy.readthedocs.io/en/stable/kinds_of_types.html#tuple-types